spec/core_functions/color/legacy.hrx

```diff
@@ -0,0 +1,182 @@
+
+<===> hsl/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> srgb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(srgb(115, 20, 210))}
+
+<===> srgb/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> srgb-linear/input.scss
+@use "sass:color";
+a {b: color.is-legacy(?????)}
+
+<===> srgb-linear/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> display-p3/input.scss
+@use "sass:color";
+a {b: color.is-legacy((display-p3 0.515 0.35 0.3 / 1))}
+
+<===> display-p3/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> a98-rgb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(?????)}
+
+<===> a98-rgb/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> prophoto-rgb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(?????)}
+
+<===> prophoto-rgb/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> rec2020/input.scss
+@use "sass:color";
+a {b: color.is-legacy(?????)}
+
+<===> rec2020/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> xyz/input.scss
+@use "sass:color";
+a {b: color.is-legacy(xyz(0.0426 0.0442 0.0364))}
+
+<===> xyz/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> xyz-d50/input.scss
+@use "sass:color";
+a {b: color.is-legacy(?????)}
+
+<===> xyz-d50/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> xyz-d65/input.scss
+@use "sass:color";
+a {b: color.is-legacy(?????)}
+
+<===> xyz-d65/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> lab/input.scss
+@use "sass:color";
+a {b: color.is-legacy(lab(5 110 115))}
+
+<===> lab/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> lch/input.scss
+@use "sass:color";
+a {b: color.is-legacy(lch(90.6 52.8 197))}
+
+<===> lch/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> oklab/input.scss
+@use "sass:color";
+a {b: color.is-legacy(oklab(0.44027, 0.08818, -0.13386))}
+
+<===> oklab/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> oklch/input.scss
+@use "sass:color";
+a {b: color.is-legacy(oklch(70% 0.1 200))}
+
+<===> oklch/output.css
+a {
+  b: false;
+}
+
+<===>
```

Comment:
@SondraE I think this is the beginning of a new file so line 182 here can be removed (unless you are adding more of course).